app/.babelrc

```diff
@@ -4,5 +4,12 @@
   ],
   "plugins" : [
     "transform-flow-comments"
-  ]
+  ],
+  "env": {
+    "test": {
+      "plugins": [
+        "dynamic-import-node"
```

Comment:
why only in test?

Reply:
Tried it for all environments and.. code splitting stopped working. Apparently this plugin does not let webpack do its job. Looks like with it `import()` is more like `require()`.

Reply:
And I added it for tests cause I aimed pretty much at `node` itself, not our webpack bundle.